test/unit/test_system_client.py

```diff
@@ -133,3 +133,43 @@ def test__ok(self):
         out, err = dbt.clients.system.run_cmd(self.run_dir, self.exists_cmd)
         self.assertEqual(out.strip(), b'hello')
         self.assertEqual(err.strip(), b'')
+
+
+class TestFindMatching(unittest.TestCase):
+
+    def setUp(self):
+        self.base_dir = '/tmp'
```

Comment:
This is failing the Windows tests, because `/tmp` isn't valid there. I think a better approach is to set `self.base_dir = mkdtemp()`. Everything else should work out nicely.

Then also define a `tearDown` method here that does `shutil.rmtree(self.base_dir)`.